### DISCUSSION TITLE
/entrypoint.sh: line 77: /data/mumble_server_config.ini: Permission Denied

### DISCUSSION THREAD

**Discussion by trymeouteh** [mumble-voip/mumble-docker#35] [Q&A] [answered]
When I run the container in podman, I get this error in my logs which is preventing me from running the container.

`/entrypoint.sh: line 77: /data/mumble_server_config.ini: Permission Denied`

This is the command I use to run the container

`podman run --detach --name mumble-server --publish 64738:64738/tcp --publish 64738:64738/udp --volume /home/testing/Documents/MumbleServers:/data --restart on-failure mumblevoip/mumble-server`

**Comment by Krzmbrzl** [comment]
Does the issue persist when starting the server without podman?

**Reply by trymeouteh** [reply]
I does work with Docker. Not sure why it will not work in podman also.

`sudo docker run --detach --name mumble-server --publish 64738:64738/tcp --publish 64738:64738/udp --volume /home/testing/Documents/MumbleServers:/data --restart on-failure mumblevoip/mumble-server`

**Reply by Krzmbrzl** [reply]
Okay. Then the issue seems to lie somewhere with podman. I have no clue where though - never used it myself 👀

**Reply by trymeouteh** [reply]
Asking on [reddit](https://www.reddit.com/r/podman/comments/171jt2a/comment/k3tumlv/?utm_source=share&utm_medium=web2x&context=3), I got this as a working solution.

`podman run --detach --name mumble-server --publish 64738:64738/tcp --publish 64738:64738/udp --volume /home/testing/Documents/MumbleServers:/data --restart on-failure --userns=keep=id mumblevoip/mumble-server`

It seems to work, will `--userns=keep=id` be the proper way for the mumble-server container to adjust the UID and GID permissions? I did see some tweaks in the readme on changing the UID and GID for the container which is the reason that caused this container to not work in the first place in podman.

**Reply by Krzmbrzl** [reply]
Changing the GID and UID is done when building the image (at this point, anyway - there's an open issue about changing that). 

> will --userns=keep=id be the proper way for the mumble-server container to adjust the UID and GID permissions? 

No idea :shrug:

**Comment by bangert** [comment, marked as answer]
For anyone looking at this trying to make this work with podman: i fixed this symptom:
```
/entrypoint.sh: line 81: /data/mumble_server_config.ini: Permission Denied
```
not by adding a `--userns=keep-id` (misspelled in the parent post), but by adding a `:z` to the volume definition: `--volume /path/to/somewhere:/data:z` - which is related to the selinux config of the volume.

**Reply by Krzmbrzl** [reply]
Thanks for sharing your solution :+1:
